File: src/main/kotlin/me/zimy/questionnaire/configuration/RecipientList.java
package me.zimy.questionnaire.configuration;

import org.springframework.boot.context.properties.ConfigurationProperties;
import org.springframework.context.annotation.Configuration;

import java.util.List;

/**
 * @author Dmitriy &lt;Zimy&gt; Yakovlev
 * @since 12/27/14.
 */
@Configuration
@ConfigurationProperties(prefix = "mail")
public class RecipientList {
    List<String> recipients;

    public List<String> getRecipients() {
        return recipients;
    }

    public void setRecipients(List<String> recipients) {
        this.recipients = recipients;
    }
}

File: src/main/kotlin/me/zimy/questionnaire/configuration/RecipientList.kt
package me.zimy.questionnaire.configuration

import org.springframework.boot.context.properties.ConfigurationProperties
import org.springframework.context.annotation.Configuration
import java.util.*

/**
 * @author Dmitriy &lt;Zimy&gt; Yakovlev
 * *
 * @since 12/27/14.
 */
@Configuration
@ConfigurationProperties(prefix = "mail")
open class RecipientList {
    var recipients: List<String> ? = ArrayList()
}
